### PR TITLE
Documentation for macosx

### PR DESCRIPTION
**Pull request by ashish0910**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,39 @@ To run Sugarizer Server **without Docker**, follow the step behind. Commands are
...
 * go to the neighborhood view, you should see one icon for the server and one for you.
 
+## Running on Macosx:Installation on macosx machine
+
+**Installing Node.js**
+* Go to the [Node.js Downloads page](https://nodejs.org/en/download/ "Node.js Downloads page")
+* Download Node.js for macOS by clicking the "Macintosh Installer" option
+* Run the downloaded Node.js .pkg Installer
+* Run the installer, including accepting the license, selecting the destination, and authenticating for the install.
+* To ensure Node.js has been installed, run node -v in your terminal - you should get something like v8.9.4
```
Comment (tarunsinghal92) [inline README.md]: add documentation for installing using `homebrew` & `nvm`

File: README.md [hunk 2]
```diff
@@ -77,6 +77,39 @@ To run Sugarizer Server **without Docker**, follow the step behind. Commands are
...
+
+**Installing MongoDb**
+* Download the MongoDB .tar.gz tarball from [here](https://www.mongodb.com/download-center/community?jmp=docs "here").
```
Comment (tarunsinghal92) [inline README.md]: `homebrew` docs

**Comment by ashish0910** [comment]
@tarunsinghal92 changes done

**Comment by ashish0910** [comment]
@tarunsinghal92 changed the documentation according to homebrew .

**Comment by tarunsinghal92** [comment]
lgtm!

**Pull request merged**